XenAdmin/SettingsPanels/NRPEEditPage.cs

```diff
@@ -384,13 +376,20 @@ private void CheckDataGridView_BeginEdit(object sender, DataGridViewCellCancelEv
 
         private void CheckDataGridView_EndEdit(object sender, DataGridViewCellEventArgs e)
         {
-            DataGridViewCell currentCell = CheckDataGridView.CurrentRow.Cells[e.ColumnIndex];
-            if (!IsHost && currentCell.Value.ToString().Trim().Equals(""))
+            DataGridViewCell currentCell = CheckDataGridView.CurrentRow?.Cells[e.ColumnIndex];

```

Comment:
Null check on the CurrentRow was missing. If it is null, then currentCell will be null, hence we also check below.